django_lifecycle/conditions/__init__.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+from typing import Iterable
+from typing import Union
+
+from django_lifecycle import NotSet
+from django_lifecycle.conditions.base import ChainableCondition
+
+
+__all__ = [
+    "WhenFieldWas",
+    "WhenFieldIsNow",
+    "WhenFieldHasChanged",
+    "WhenFieldIsNot",
+    "WhenFieldWasNot",
+    "WhenFieldChangesTo",
+    "Always",
+]
+
+
+@dataclass
+class WhenFieldWas(ChainableCondition):
+    field_name: str
+    was: Any = "*"
+
+    def __call__(
+        self, instance: Any, update_fields: Union[Iterable[str], None] = None
+    ) -> bool:
+        return self.was in (instance.initial_value(self.field_name), "*")
+
+
+@dataclass
+class WhenFieldIsNow(ChainableCondition):
```

Comment:
I believe the term 'now' may not add significant meaning. Similarly, what would be the appropriate name for the condition 'WhenFieldWas' or 'WhenFieldBefore'?
```suggestion
class WhenFieldIs(ChainableCondition):
```
Moreover, the term 'now' might lead to confusion. Does it represent the value at the moment when the condition is evaluated? Does it represent the value at the time the object is created?